### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by KayLemba**
In this this project I created a Todo list using  html, JavaScript and Css. I did the following:

1. Add src folder.
2. Add index.html file
3. Add index.js file 
4. Add style.css file
5. Add README.md file.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+});
+
+const listSection = document.querySelector('#list-items');
+listSection.innerHTML = '';
+for (let i = 0; i < todoList.length; i += 1) {
+  const task = todoList[i];
+  let taskItem = `
+    <li class="d-flex s-between list-item">`;
+  if (task.completed) {
+    taskItem += `<span class="material-icons done" onclick="updateStatus(${task.index}, 'pending')">
```
Comment (henatan99) [inline src/index.js:54]: - [ ]  Whenever I click the check box, it throws a console error concerning the `onClick` event handler `updateStatus(${task.index}, 'pending')`. Kindly fix this error. :pray: 

Screenshot from 2021-11-30 22-46-47

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @KayLemba , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (KayLemba) [reply src/index.js:54]: Hello @henatan99 , Kindly note that the requested change is not part of the requirements. I was trying to implement an onlcick event that I will use in the next project activity as per requirement.

Comment (KayLemba) [reply src/index.js:54]: However, I have fixed the issue.

Comment (KayLemba) [reply src/index.js:54]: However, I have fixed the issue.

**Review by Lameck1 — APPROVED** [review]

**Review by Lameck1 — APPROVED** [review]
Hi, @KayLemba,

The tasks for this milestone have been successfully implemented 💯.
All the best with the next milestone.

You are free to merge this branch :shipit:

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**